api/controller/EducationController.py

```diff
@@ -12,15 +12,17 @@ def get_education(self, id):
             education = Education.objects.get(id=id)
             return {
                 "data": {
-                    "type": str(education),
-                    "id": str(education.id),
-                    "attributes": {
-                        "classification": education.classification,
-                        "question": education.question,
-                        "description": education.description,
-                        "information": education.information,
-                        "note": education.note,
-                        "source": education.source
+                    "01_type": str(education),
+                    "02_id": str(education.id),
+                    "03_attributes": {
+                        "A_order": education.order,
+                        "B_classification": education.classification,
+                        "C_question": education.question,
+                        "D_description": education.description,
+                        "E_information": education.information,
+                        "F_note": education.note,
+                        "G_source": education.source,
+                        "H_symbol": education.symbol
```

Comment:
Thanks for alphabetizing these, definitely nice to not have to sort on our end :)